src/EditorFeatures/CSharp/UseAutoProperty/UseAutoPropertyAnalyzer.cs

```diff
@@ -81,11 +83,16 @@ private bool CheckExpressionSyntactically(ExpressionSyntax expression)
         protected override ExpressionSyntax GetGetterExpression(IMethodSymbol getMethod, CancellationToken cancellationToken)
         {
             var getAccessor = getMethod.DeclaringSyntaxReferences[0].GetSyntax(cancellationToken) as AccessorDeclarationSyntax;
-            var firstStatement = getAccessor?.Body.Statements.SingleOrDefault();
-            if (firstStatement?.Kind() == SyntaxKind.ReturnStatement)
+            var statements = getAccessor?.Body?.Statements ?? SpecializedCollections.EmptyEnumerable<StatementSyntax>();
```

Comment:
A getter body's `Statements` property can be null, like in this case: `int P { get; set; }`.